### ISSUE TITLE
Pytorch lightning?

### ISSUE THREAD

**Issue by reactivetype** [karanchahal/nyu-cv-project#1]
Very thorough and useful work! 

I have not seen pytorch-lightning neither in the requirement.txt nor in the code. 

Do you have a plan to refactor it to use lightning trainer and module?

**Comment by reactivetype** [comment]
I just saw you are working on it on a different branch. Btw do you welcome PRs?

**Comment by karanchahal** [comment]
Yes ! We would welcome PR's that help this repo. It could be anything, some ideas are:

1. New Distillation techniques
2. Clean up on code.

The lightning code is in another branch, presently there are no plans to merge it as of now.
